app/presenters/vendor_api/multiple_applications_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module VendorAPI
+  class MultipleApplicationsPresenter < Base
```

Comment:
Can we retrieve the collection as part of the ApplicationPresenter (by adding a new class method e.g. `ApplicationPresenter.decorated_serialised_collection(version, applications, options)` instead of introducing a new presenter?